Applications/Utils/MeshEdit/AddTopLayer.cpp

```diff
@@ -59,17 +50,23 @@ int main (int argc, char* argv[])
 	cmd.parse(argc, argv);
 
 	INFO("Reading mesh \"%s\" ... ", mesh_arg.getValue().c_str());
-	MeshLib::Mesh * subsfc_mesh(FileIO::readMeshFromFile(mesh_arg.getValue()));
+	auto subsfc_mesh = std::unique_ptr<MeshLib::Mesh>(
+	    FileIO::readMeshFromFile(mesh_arg.getValue()));
```

Comment:
I guess you are referring to the `=`. I think, there is no allocation in this `=` construct. AFAIK if you write something like `Type var = Type(...)` the C++ standard says that this is the same as `Type var(...)`. So this line should automatically behave as you suggest.